apps/flame-defi/app/features/evm-wallet/services/non-fungible-position-service/index.ts

```diff
@@ -0,0 +1,258 @@
+import type { Config } from "@wagmi/core";
+import { type Address, Abi } from "viem";
+import { HexString } from "@repo/flame-types";
+import { GenericContractService } from "../generic-contract-service";
+import NON_FUNGIBLE_POSITION_MANAGER_ABI from "./non-fungible-position-manager-abi.json";
+import { PoolPositionResponse } from "pool/types";
+
+type PoolPosition = {
```

Comment:
There is another `PoolPosition` type already.

This type appears to be `PoolPositionResponse`. If you refactor `PoolPositionResponse` to remove `tokenId` like I mention in another comment, then I don't think you need this type.